terraform/environments/data-platform/iam.tf

```diff
@@ -550,6 +550,26 @@ data "aws_iam_policy_document" "iam_policy_document_for_reload_data_product_lamb
   }
 }
 
+data "aws_iam_policy_document" "iam_policy_document_for_get_schema_lambda" {
+  statement {
+    sid    = "s3LogAccess"
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+    ]
+    resources = [
+      "${module.s3-bucket.bucket.arn}/logs/*"
```

Comment:
You will need to change this to use the new separated buckets. This lambda should have read access to the metadata bucket.

I think we can remove module.s3-bucket soon, just need to check there is nothing in it we want to keep.